packages/@aws-cdk/aws-appmesh/lib/tls-validation.ts

```diff
@@ -1,33 +1,36 @@
 import * as acmpca from '@aws-cdk/aws-acmpca';
-import { CfnVirtualGateway, CfnVirtualNode } from './appmesh.generated';
+import { CfnVirtualNode } from './appmesh.generated';
 
 // keep this import separate from other imports to reduce chance for merge conflicts with v2-main
 // eslint-disable-next-line no-duplicate-imports, import/order
 import { Construct } from '@aws-cdk/core';
 
 /**
- * Represents the properties needed to define TLS validation context
+ * Represents the properties needed to define TLS Validation context
  */
 export interface TlsValidation {
+  /**
+   * Represents the subject alternative names (SANs) secured by the certificate.
+   * SANs must be in the FQDN or URI format.
+   *
+   * @default - the Envoy proxy for that node doesn't verify the SAN on a peer client certificate.
+   */
+  readonly subjectAlternativeNames?: SubjectiveAlternativeNames;
+
   /**
    * Reference to where to retrieve the trust chain.
    */
   readonly trust: TlsValidationTrust;
 }
 
 /**
- * All Properties for TLS Validations for both Client Policy and Listener.
+ * All Properties for TLS Validation Trusts for both Client Policy and Listener.
  */
 export interface TlsValidationTrustConfig {
   /**
-   * VirtualNode CFN configuration for client policy's TLS Validation
-   */
-  readonly virtualNodeClientTlsValidationTrust: CfnVirtualNode.TlsValidationContextTrustProperty;
-
-  /**
-   * VirtualGateway CFN configuration for client policy's TLS Validation
+   * VirtualNode CFN configuration for client policy's TLS Validation Trust
    */
-  readonly virtualGatewayClientTlsValidationTrust: CfnVirtualGateway.VirtualGatewayTlsValidationContextTrustProperty;
+  readonly tlsValidationTrust: CfnVirtualNode.TlsValidationContextTrustProperty;
```

Comment:
Can you explain the motivation for this change?

Reply:
Sorry, I should have mentioned this change in the commit message.

The motivation is simply reducing the repetition. All four properties share exactly same shape and thus reduced into single property.

The updated modeling is mimicking [`tls-certificate.ts`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-appmesh/lib/tls-certificate.ts).